configs/opencv-ovms/models/2022/config_template.json

```diff
@@ -33,6 +33,17 @@
                 "batch_size":"1",
                 "plugin_config": {"PERFORMANCE_HINT": "LATENCY"},
                 "target_device": "{target_device}"}
+            },
+            {"config": {
+                "name": "person_vehicle_bike_detection_2000",
+                "base_path": "/models/person_vehicle_bike_detection_2000/FP16-INT8",
+                "nireq": 1,
+                "batch_size":"1",
+		"shape": "auto",
+                "layout": "NHWC:NCHW",
+                "plugin_config": {"PERFORMANCE_HINT": "LATENCY"},
+                "target_device": "CPU"},
```

Comment:
```suggestion
                "target_device": "{target_device}"},
```

Reply:
Should we use {target_device} or "MULTI:GPU,CPU" ? 
where does the value of target_device come from?

Reply:
it is the placeholder for the script to replace it upon user's input in command line

Reply:
you probably missed this PR from Brian: https://github.com/intel-retail/automated-self-checkout/pull/209  check it out